Header header
geometry_msgs/Vector3 position
geometry_msgs/Vector3 scale
string obj_class
float32 score
